vendor/assets/stylesheets/susy/susy/language/susy/_background.scss

```diff
@@ -0,0 +1,386 @@
+  #{$_vert}: 10px;
+  z-index: 999;
+  color: #333;
+  background: rgba(white, .25);
+}
+
+
+// [Private] Grid Overlay Base
+// ---------------------------
+// Base styles for generating a grid overlay
+@mixin grid-overlay-base() {
+  position: absolute;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  content: " ";
+  z-index: 998;
+}
+
+
+// Get Symmetrical Background
+// --------------------------
+// - $grid: <map>
+@function get-background-sym(
+  $grid
+) {
+  $grid           : parse-grid($grid);
+  $_gutters       : susy-get(gutters, $grid);
+  $_column-width  : susy-get(column-width, $grid);
+  $_math          : susy-get(math, $grid);
+
+  $_color         : debug-get(color);
+  $_trans         : transparent;
+  $_light         : lighten($_color, 15%);
+
+  $_end           : 1 + $_gutters;
+  $_after         : percentage(1/$_end);
+  $_stops         : ();
+  $_size          : span(1 $grid wide);
+
+  @if is-inside($grid) {
+    $_stops: $_color, $_light;
+  } @else if is-split($grid) {
+    $_split: $_gutters/2;
+    $_before: percentage($_split/$_end);
+    $_after: percentage((1 + $_split)/$_end);
+    $_stops: $_trans $_before, $_color $_before, $_light $_after, $_trans $_after;
+  } @else {
+    $_stops: $_color, $_light $_after, $_trans $_after;
+  }
+
+  @if $_math == static {
+    $_size: valid-column-math($_math, $_column-width) * $_end;
+  }
+
+  $_output: (
+    image: (columns: $_stops),
+    size: $_size,
+  );
+
+  @return $_output;
+}
+
+
+// Get Asymmetrical Inside
+// -----------------------
+// - $grid: <settings>
+@function get-asym-inside(
+  $grid
+) {
+  $grid     : parse-grid($grid);
+  $_columns : susy-get(columns, $grid);
+
+  $_color   : debug-get(color);
+  $_light   : lighten($_color, 15%);
+  $_stops   : ();
+
+  @for $location from 1 through susy-count($_columns) {
+    $this-stop: ();
+
+    @if $location == 1 {
+      $this-stop: append($this-stop, $_color, comma);
+    } @else {
+      $start: parse-span(1 at $location $grid);
+      $start: get-isolation($start);
+      $this-stop: append($this-stop, $_color $start, comma);
+    }
+
+    @if $location == susy-count($_columns) {
+      $this-stop: append($this-stop, $_light, comma);
+    } @else {
+      $_end: parse-span(1 at ($location + 1) $grid);
+      $_end: get-isolation($_end);
+      $this-stop: append($this-stop, $_light $_end, comma);
+    }
+
+    $_stops: join($_stops, $this-stop, comma);
+  }
+
+  @return $_stops;
+}
+
+
+// Get Asymmetrical Split
+// ----------------------
+// - $grid: <settings>
+@function get-asym-split(
+  $grid
+) {
+  $grid     : parse-grid($grid);
+  $_columns : susy-get(columns, $grid);
+
+  $_color   : debug-get(color);
+  $_light   : lighten($_color, 15%);
+  $_stops   : ();
+
+  @for $location from 1 through susy-count($_columns) {
+    $this-stop: ();
+
+    $start: parse-span(1 at $location $grid);
+    $start: get-isolation($start);
+    $this-stop: append($this-stop, transparent $start, comma);
```

Comment:
Color literals like `transparent` should only be used in variable declarations; they should be referred to via variable everywhere else.